src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2456,15 +2453,15 @@ static void sub_6CBCE2(
         // Set the temporary track element
         _tempTrackTileElement.SetType(TILE_ELEMENT_TYPE_TRACK);
         _tempTrackTileElement.SetDirection(trackDirection);
-        _tempTrackTileElement.AsTrack()->SetHasChain((edx & 0x10000) != 0);
+        _tempTrackTileElement.AsTrack()->SetHasChain((liftHillAndInvertedState & CONSTRUCTION_LIFT_HILL_SELECTED) != 0);
```

Comment:
You changed the value here from 0x10000 to 0x1. Are all callers of this function also modified?

Reply:
yes thats the point of the change.

Reply:
I didn't see any changed callers to `sub_6CBCE2`, so I was wondering if it was complete.

Reply:
`window_ride_construction_draw_track_piece` is the only caller and it used to stuff a load of unused information in that var as well. I've removed the unneeded information and removed the << 16.